rll/upgrade.sh

```diff
@@ -20,18 +20,21 @@
 
 UPGRADES_FILE_LOCATION=${UPGRADES_FILE_LOCATION:-"https://rightlink.rightscale.com/rightlink/upgrades"}
 
+# Determine base directory of rightlink / rsc
```

Comment:
With Linux paths, the components are often referred to as the 'directory name' (dirname), and 'basename'.  Perhaps:
`# Determine directory location of rightlink / rsc`